### PR TITLE
e2e: enclave-cc updates

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

File: tests/e2e/kind/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ARG IMAGE=kindest/node
-ARG VERSION=1.26
-ARG MINOR=0
+ARG VERSION=1.27
```
Comment (wainersm) [inline tests/e2e/kind/Dockerfile:2]: Just curious why you broke it in version and minor instead of a single version=vMAJOR.MINOR

**Review by wainersm — APPROVED** [review]

**Comment by mythi** [comment]
this is needed for the release

**Comment by fidencio** [comment]
CLH TDX passed: http://10.112.240.228:8080/job/confidential-containers-operator-main-centos8stream-x86_64-containerd_kata-clh-tdx-PR/286/console

**Pull request merged**